src/device/ping-time.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+namespace PingTime {
+  void microsecondDelay(unsigned int microseconds);
+  int timeMs();
+} // namespace PingTime
```

Comment:
this functions miss const.

Reply:
I don't understand this suggestion.

Reply:
this functions are const, the const keyword should be used.